Sage/carbon
# chore(dependancies): resolve high vulnerabilities

### Proposed behaviour

Resolve high vulnerabilities flagged by `npm audit` 

### Current behaviour

Currently 10 high vulnerability warnings are displayed when running `npm audit`

### Checklist


<del> - [ ] Screenshots are included in the PR </del>
<del> - [ ] Carbon implementation and Design System documentation are congruent </del>
<del> - [ ] All themes are supported </del>
<del> - [ ] Unit tests added or updated </del>
<del> - [ ] Cypress automation tests added or updated </del>
<del> - [ ] Storybook added or updated </del>
<del> - [ ] Typescript `d.ts` file added or updated </del>

### Additional context
N/A

### Testing instructions
- Pull down the branch
- run `npm audit`
- Message should appear in the console "found 15 vulnerabilities (12 low, 1 moderate, 2 high) in 4488 scanned packages".
